File: Backend/Go/Domain/Model/CustomModel/CompleteHero.go
package domain_model_custom

import (
	domain_model "github.com/almiranda86/superhero-go/Domain/Model"
	domain_model_base "github.com/almiranda86/superhero-go/Domain/Model/BaseModel"
)

type CompleteHero struct {
	Powerstats     domain_model.Powerstats
	Biography      domain_model.Biography
	Appearance     domain_model.Appearance
	Work           domain_model.WorK
	Connections    domain_model.Connections
	Image          domain_model.Image
	BaseProperties domain_model_base.BaseModel
}

File: Backend/C#/SuperHero.Domain/Model/CustomModel/CompleteHero.cs
using MongoDB.Bson.Serialization.Attributes;
using MongoDB.Bson;
using SuperHero.Domain.Model.BaseModel;
using System.Text.Json.Serialization;

namespace SuperHero.Domain.Model.CustomModel
{
    public class CompleteHero : IGuidId
    {
        [JsonPropertyName("powerstats")]
        public Powerstats Powerstats { get; set; }

        [JsonPropertyName("biography")]
        public Biography Biography { get; set; }

        [JsonPropertyName("appearance")]
        public Appearance Appearance { get; set; }

        [JsonPropertyName("work")]
        public Work Work { get; set; }

        [JsonPropertyName("connections")]
        public Connections Connections { get; set; }

        [JsonPropertyName("image")]
        public Image Image { get; set; }

        [JsonPropertyName("name")]
        public string Name { get; set; }

        [BsonId]
        [JsonPropertyName("public_id")]
        public string PublicId { get; set; }

    }
}
